seatunnel-e2e/seatunnel-connector-v2-e2e/connector-kudu-e2e/src/test/java/org/apache/seatunnel/e2e/connector/kudu/KuduWIthMultipleTableIT.java

```diff
@@ -210,10 +215,6 @@ private void initializeKuduTable() throws KuduException {
                 new ColumnSchema.ColumnSchemaBuilder("val_unixtime_micros", Type.UNIXTIME_MICROS)
                         .nullable(true)
                         .build());
-        columns.add(
```

Comment:
Why remove `binary` type?

Reply:
binary type, difficult to generate in fake, and difficult to compare,KuduIT contains all types of checks